### ISSUE TITLE
Distance & Average Speed are way off…

### ISSUE THREAD

**Issue by mohs8421** [flyingrub/SpeedMeter#30]
This morning I tried Speedmeter while cycling to work. The current speed seemed to be accurate in the most cases (difference to actual cycling computer usually 1km/h).
I travelled my usual 8 km to work, but the distance it said was 12.363 km. The average speed was 28 km/h because of this, the actual average was 19.55 km/h.

It stated the Accuracy would be 2-5m most of the time.

**Comment by flyingrub** [comment]
We try to avoid the point where we move less than the current accuracy https://github.com/flyingrub/SpeedMeter/blob/master/app/src/main/java/fly/speedmeter/grub/GpsServices.java#L63.
Also we get a location update every 500ms : https://github.com/flyingrub/SpeedMeter/blob/master/app/src/main/java/fly/speedmeter/grub/GpsServices.java#L44.
We can perhaps tweak this value to 1s or more to avoid having a lot of inaccurate point.

**Comment by AlexDaniel** [comment]
Yes, this is a real problem. Just traveled 3 km and it showed 9.9 km :( Too inaccurate to be useful.
